README.md

```diff
@@ -18,12 +18,20 @@ Because [tchannel-node](https://github.com/uber/tchannel-node) does not have ins
 for OpenTracing, Jaeger-Client exposes methods wrapping tchannel handlers, and encoded channels.
 An encoded channel is a channel wrapped in either a thrift encoder `TChannelAsThrift`,
 or json encoder `TChannelAsJson`.  To wrap a server handler for thrift one can initialize
-a tchannel bridge, and wrap there encoded handler function with `tracedHandler` decorator.
+a tchannel bridge, and wrap the encoded handler function with a `tracedHandler` decorator.
+The tchannel bridge takes an OpenTracing  tracer, and a context factory.  The context factory
+must be a function that returns a context with the methods 'getSpan', and 'setSpan' which retrieve 
+and assign the span to the context respectively.
```

Comment:
fwiw, you can even relax the requirement for getSpan/setSpan methods by taking respective functions as constructor arguments. It would allow people to use a context object that simply has a field for a span, rather than methods. E.g.
```
function getSpan(context: Context): opentracing.Span
function setSpan(context: Context, span: opentracing.Span): void
```